front-end/src/components/User.js

```diff
@@ -30,10 +30,11 @@ const useStyles = makeStyles((theme) => ({
   },
 }));
 
-const User = ({ username }, { userId }) => {
+const User = (props) => {
   const classes = useStyles();
   const [secrets, setSecrets] = useState([]);
   const [secretUpdate, setSecretUpdate] = useState(0);
+  const { userId, username } = props;
```

Comment:
Even though this doesn't change anything functionally, it did get rid of both the linting "react/no-unused-prop-types" false positives.